source/extensions/geoip_providers/maxmind/BUILD

```diff
@@ -34,6 +35,7 @@ envoy_cc_library(
     name = "provider_impl",
     srcs = select({
         "//bazel:linux": ["geoip_provider.cc"],
+        "//bazel:apple": ["geoip_provider.cc"],
```

Comment:
I think `//bazel:apple` will also include iOS (for envoy-mobile). Is that intended?

Reply:
@ggreenway that was not intended, will fix and retest